sourcemash/frontend/templates/splash.tpl

```diff
@@ -0,0 +1,6 @@
+<div class="row">
+  <div class="col s12 valign-wrapper">
+    <img src="/static/solologo.svg">
+    <img src="/static/logotext.svg">
+  </div>
+</div>
```

Comment:
That requires cleaning up the image and making it transparent & figuring out how to crop SVG files and/or enhance PNG files. I raise UI Design updates as an issue for next week, and figured this is just a placeholder until we actually have a splash page to land on.